Frontend/src/test/pages/cart/Cart.test.jsx

```
@@ -0,0 +1,148 @@
+    error: vi.fn(),
+  },
+}));
+
+import { toast } from 'react-hot-toast';
+import { navigateMock } from 'react-router-dom';
+import * as CartContex from '../../../pages/cart/context/CartContex';
+import Cart from '../../../pages/cart/Cart';
+
+// Mock UI and child components
+vi.mock('../../../components/ui/button', () => ({
+  Button: ({ children, onClick }) => <button onClick={onClick}>{children}</button>,
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)